src/pretalx/orga/templates/orga/auth/base.html

```diff
@@ -20,7 +20,7 @@
     <body>
         <div id="wrapper"{% if request.event.primary_color %} style="background-color: {{ request.event.primary_color }}"{% endif %}>
             {% if not request.event.logo %}
-                <img src="{% static "common/img/logo_white.svg" %}" class="logo" alt="{% translate "The pretalx logo" %}">
+                <img src="{% static "common/img/logo_white.png" %}" class="logo" alt="{% translate "The eventyay logo" %}">
```

Comment:
I think no need to mention eventyay here. We are aiming for a whitelabel solution. In future I guess we also need a configuration for the logos, but it is maybe out of scope of this PR.